clients/src/test/java/org/apache/kafka/test/TestUtils.java

```diff
@@ -178,4 +184,66 @@ public static Properties producerConfig(final String bootstrapServers,
         properties.putAll(additional);
         return properties;
     }
+
+    // provides producer properties with default String serializers for keys and values
+    public static Properties producerConfigStringKeysValues(final String bootstrapServers) {
```

Comment:
> Any specific reasons that we want to have a direct constructor for string and integer serdes?

@guozhangwang 
Just trying to eliminate key strokes for some of the more commonly used serdes.  I'm not married to the idea, so I'll do as you suggest.